samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -14,13 +14,17 @@
 #define DEVICE_NAME CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
 #define PEER_NAME_LEN_MAX 30
-#define TIMEOUT_SYNC_CREATE K_SECONDS(10)
+/* BT Core 5.3 specification allows controller to wait 6 periodic advertising events for
+ * synchronization establishment, hence timeout must be longer than that.
+ */
+#define SYNC_CREATE_TIMEOUT_INTERVAL_NUM 7
 
 static struct bt_le_per_adv_sync *sync;
 static bt_addr_le_t per_addr;
 static volatile bool per_adv_found;
 static bool scan_enabled;
 static uint8_t per_sid;
+static uint16_t sync_create_timeout_ms;
```

Comment:
This value could overflow for long advertising intervals I think (if I recall correctly the advertising interval could be even up to 10.24 seconds). Maybe we could make it uin32_t to ensure that this would never happen?

Reply:
True, changed to `uint32_t`